.github/workflows/deploy_docs.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     tags:
       - '*'
+  pull_request:
```

Comment:
Is this here for testing? Or do you need to include the target branch name the PR is merging to?

Reply:
Well, when I cut the release yesterday, publishing "javadoc" was broken (because we don't run it in CI). So now I'm running both forms of generating docs on every PR.